apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Beregningsgrunnlag.tsx

```diff
@@ -17,27 +17,50 @@ import { isPending, isSuccess } from '~shared/api/apiUtils'
 import { useFeatureEnabledMedDefault } from '~shared/hooks/useFeatureToggle'
 import styled from 'styled-components'
 import { isFailureHandler } from '~shared/api/IsFailureHandler'
+import { statusErRedigerbar } from '~components/behandling/felles/utils'
 
 const Beregningsgrunnlag = (props: { behandling: IDetaljertBehandling }) => {
   const { behandling } = props
 
-  const [overstyrtBeregning, getOverstyrtBeregning] = useApiCall(hentOverstyrBeregning)
-  const [overstyrt, setOverstyrt] = useState<OverstyrBeregning | undefined>(undefined)
+  const [overstyrtBeregningGrunnlagRest, getOverstyrBeregningGrunnlag] = useApiCall(hentOverstyrBeregningGrunnlag)
```

Comment:
Usikker på om det er `...Rest`. Men mulig det ikke er gjort konsistent eller heller.

Reply:
Ikke gjort konsistent, men la til for å synliggjøre bedre hva som er api responsen, og hva som er data objektet ettersom vi sjekker begge. Har renamet til `...Response` fremfor `...Rest`